Samples/src/Arithmetic/Adder Example.ipynb

```diff
@@ -0,0 +1,232 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Integer arithmetic\n",
+    "\n",
+    "Many quantum algorithms need to evaluate classical functions on a superposition of inputs. An example is Shor's algorithm, which requires the mapping $|x\\rangle|0\\rangle\\mapsto|x\\rangle|a^x\\text{ mod }N\\rangle$ to be implemented as a quantum operation.\n",
+    "\n",
+    "To keep users from having to implement such functions from scratch, the Canon (`Microsoft.Quantum.Canon`) contains arithmetic functionality such as integer adders."
```

Comment:
We've generally used a lowercase "canon" in these cases. It may also be reasonable to link to either the API or conceptual docs (https://docs.microsoft.com/qsharp/api/canon/microsoft.quantum.canon or https://docs.microsoft.com/quantum/libraries/standard/, respectively).

Reply:
Done